authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc

```diff
@@ -20,7 +20,7 @@ For information about configuring your cloud provider to use manual mode, see th
 [id="manual-mode-sts_{context}"]
 == Manual mode with cloud credentials created and managed outside of the cluster
 
-An AWS or GCP cluster that uses manual mode might be configured to create and manage cloud credentials from outside of the cluster using the AWS Security Token Service (STS) or GCP Workload Identity. With this configuration, the CCO uses short-term credentials for different components.
+AWS, GCP, and global Azure clusters that uses manual mode might be configured to create and manage cloud credentials from outside of the cluster. This credentials management strategy uses the AWS Security Token Service (STS), GCP Workload Identity, or Azure AD Workload Identity. In this configuration, the CCO uses temporary credentials for different components.
```

Comment:
For the clash, I will defer to the formatting in #64848 - I think having this as a note makes more sense than a subheading since the page it's on is for long-term manual creds. Pushing a commit with new wording, lmk what you think. If it works here, I'll put it in the other PR as well :)